test/destroyed_at_test.rb

```diff
@@ -260,3 +260,11 @@
   end
 end
 
+describe 'destroying on object should call after_commit callback' do
+  it 'calls after_commit callback on: :destroy' do
+    comment = Comment.create
+    comment.destroy
+
+    comment.after_committed.must_equal true
```

Comment:
hmm, but I have `after_commit` with `on: :destroy` option. It shouldn't be called on create, should it?